packages/vats/src/core/types-ambient.d.ts

```diff
@@ -232,6 +232,7 @@ type WellKnownContracts = {
   provisionPool: typeof import('@agoric/inter-protocol/src/provisionPool.js').start;
   priceAggregator: typeof import('@agoric/inter-protocol/src/price/fluxAggregatorContract.js').start;
   reserve: typeof import('@agoric/inter-protocol/src/reserve/assetReserve.js').start;
+  reserveGovernor: typeof import('@agoric/governance/src/contractGovernor.js').start;
```

Comment:
redundant with `contractGovernor`?

Reply:
no because this is another entry in the space. it just happens to have the same type as contractGovernor